WordPress/WordPressTest/Classes/Stores/StatsWidgetsStoreTests.swift

```diff
@@ -28,7 +29,7 @@ class StatsWidgetsStoreTests: CoreDataTestCase {
         XCTAssertTrue(statsWidgetsHaveData())
     }
 
-    func testStatsWidgetsDeletedAfterDefaultWPAccountRemoved() {
+    func xTestStatsWidgetsDeletedAfterDefaultWPAccountRemoved() {
```

Comment:
I think making tests `private` disables them. Alternatively and maybe clearer in the test output, we could `XCTSkip` them.

```suggestion
    private func testStatsWidgetsDeletedAfterDefaultWPAccountRemoved() {
```